### ISSUE TITLE
[libGDX] Java 17 requirement for `4.2.6` release seems to be too strict

### ISSUE THREAD

**Issue by fgnm** [EsotericSoftware/spine-runtimes#2597]
Hi!
I was upgrading my dependencies and I've found that `spine-libgdx` now requires Java 17. Currently is a problem for all my libGDX projects. The engine (HyperLap2D) is still forced to be compatible with Java 7 because the MobiVM, and so also modules of my games are forced between Java 11 and Java 7. I always had no troubles in the past, but with  `4.2.6`  gradle starts to get angry:

```
> Could not resolve all files for configuration ':h2d-libgdx-spine-extension:compileClasspath'.
   > Could not resolve com.esotericsoftware.spine:spine-libgdx:4.2.6.
     Required by:
         project :h2d-libgdx-spine-extension
      > No matching variant of com.esotericsoftware.spine:spine-libgdx:4.2.6 was found. The consumer was configured to find an API of a library compatible with Java 7, preferably in the form of class files, preferably optimized for standard JVMs, and its dependencies declared externally but:
          - Variant 'apiElements' capability com.esotericsoftware.spine:spine-libgdx:4.2.6 declares an API of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 17 and the consumer needed a component compatible with Java 7
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
          - Variant 'runtimeElements' capability com.esotericsoftware.spine:spine-libgdx:4.2.6 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 17 and the consumer needed a component compatible with Java 7
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
```

And in the games:
```
> Could not resolve all files for configuration ':core:compileClasspath'.
   > Could not resolve com.esotericsoftware.spine:spine-libgdx:4.2.6.
     Required by:
         project :core
      > No matching variant of com.esotericsoftware.spine:spine-libgdx:4.2.6 was found. The consumer was configured to find a library for use during compile-time, compatible with Java 11, preferably in the form of class files, preferably optimized for standard JVMs, and its dependencies declared externally but:
          - Variant 'apiElements' capability com.esotericsoftware.spine:spine-libgdx:4.2.6 declares a library for use during compile-time, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component, compatible with Java 17 and the consumer needed a component, compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
          - Variant 'runtimeElements' capability com.esotericsoftware.spine:spine-libgdx:4.2.6 declares a library for use during runtime, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component, compatible with Java 17 and the consumer needed a component, compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
```

The last working runtime is the `4.2.0`

**Comment by badlogic** [comment]
Sorry for that. I'll see if I can lower that requirement.

**Comment by badlogic** [comment]
I just released 4.2.7 which lowers the requirement to Java 7 again.
